lib/validation/validator.js

```diff
@@ -78,8 +78,19 @@ class Validator {
 
 const validator = Object.create(null);
 
+async function _validate(schemaName, options) {
+	if (!validator[schemaName]) {
+		validator[schemaName] = await (async () => {
```

Comment:
I think this still has potential for a race condition. 

The map entry is only set once the promise has resolved. Which is asynchronous. A second validate call might see the map as empty and start creating a new Validator instance itself.

Instead we should store the promise in the map and await on that.

Reply:
Hmmmm, I got it. I'll do it with a promise. Thanks!

Reply:
I think now it should be ok

Reply:
Looks good 👌